src/main/java/com/arpnetworking/metrics/common/kafka/RunnableConsumerImpl.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2019 Dropbox.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics.common.kafka;
+
+import com.arpnetworking.commons.builder.OvalBuilder;
+import com.arpnetworking.steno.Logger;
+import com.arpnetworking.steno.LoggerFactory;
+import net.sf.oval.constraint.NotNull;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+
+import java.time.Duration;
+
+/**
+ * A runnable wrapper for a <code>Consumer</code> that will continually poll
```

Comment:
nit: Please use `{@code ...}` or (for in-project references) `{@link ...}` instead of `<code>...</code>`.